pipelines.adoc

```diff
@@ -39,25 +39,25 @@ By default, the pipelines run and deploy the application in the `+kabanero+` nam
 
 For more information, see link:https://github.com/kabanero-io/kabanero-pipelines[the kabanero-pipelines repo].
 
-== Running Kabanero pipelines
+== Running Pipelines
 
-Explore how to use Kabanero Pipelines to build and manage Kabanero collections.
+Explore how to use Pipelines to build and manage Collections.
 
 === Prerequisites
 
-. link:https://github.com/kabanero-io/kabanero-foundation[Kabanero foundation] must be installed on Red Hat Origin Community Distribution of Kubernetes (OKD) or the OpenShift Container Platform (OCP) cluster.
+. link:https://github.com/kabanero-io/kabanero-foundation[Kabanero foundation] must be installed on Red Hat Origin Community Distribution of Kubernetes (OKD) or the OpenShift Container Platform (OCP) cluster.  At present Kabanero 0.3.0 and greater only supports OCP 4.2+.  When OKD provides a comparable 4.x stream, Kabanero will again support OKD providing a full open source stack.
```

Comment:
edit to remove future tense and double space

**change**
cluster.  At present Kabanero 0.3.0 and greater only supports OCP 4.2+.  When OKD provides a comparable 4.x stream, Kabanero will again support OKD providing a full open source stack.

**to**
cluster. Presently, Kabanero 0.3.0 and greater supports only OCP 4.2+. When OKD provides a comparable 4.x stream, Kabanero is projected to again support OKD, providing a full open source stack.